packages/flutter_tools/lib/src/base/pub_spec_content.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:yaml/yaml.dart';
+
+class PubContent {
```

Comment:
nit. let's call this `PubspecContent`, since pub is the name of the tool. Also, can you name this file `pubspec_content.dart`?

```suggestion
class PubspecContent {
```